charts/hono/templates/kafka/kafka-example-keys.yaml

```diff
@@ -14,7 +14,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  {{- $args := dict "dot" . "component" "kafka" "name" "kafka-example-keys" }}
+  {{- $args := dict "dot" (. | deepCopy | merge (dict "Values" (pick .Values.kafka "nameOverride" "fullnameOverride"))) "component" "kafka" "name" "kafka-example-keys" }}
```

Comment:
Corresponding to the other comment about omitting the duplicate `-kafka` part, this could be changed to:
```
  {{- /* metadata.name is set to "{fullNameOfKafkaSubChart}-example-keys" here, meaning e.g. "{ReleaseName}-kafka-example-keys", not "{ReleaseName}-hono-example-keys" */}}
  {{- $args := dict "dot" (. | deepCopy | merge (dict "Values" (pick .Values.kafka "nameOverride" "fullnameOverride"))) "component" "kafka" "name" "example-keys" }}
```